src/test/java/io/debezium/connector/yugabytedb/TestHelper.java

```diff
@@ -109,11 +157,21 @@ public static YugabyteDBConnectorConfig.LogicalDecoder decoderPlugin() {
      *
      * @return the PostgresConnection instance; never null
      */
+    // TODO: remove this function at the end of test development
     public static YugabyteDBConnection create() {
 
-        return new YugabyteDBConnection(defaultJdbcConfig());
+        // return new YugabyteDBConnection(defaultJdbcConfig());
+        return new YugabyteDBConnection(defaultJdbcConfig(CONTAINER_YSQL_HOST, CONTAINER_YSQL_PORT));
     }
 
+    public static YugabyteDBConnection createConnectionTo(String databaseName) {
+        return new YugabyteDBConnection(defaultJdbcConfig(CONTAINER_YSQL_HOST, CONTAINER_YSQL_PORT, databaseName));
+    }
+
+    // public static YugabyteDBConnection create(String host, String ysqlPort) {
```

Comment:
Can you remove this? We can always add it later if required. Seems like a simple fn. and available in git history